src/Compilers/CSharp/Portable/CSharpCompilationOptions.cs

```diff
@@ -54,7 +55,47 @@ public sealed class CSharpCompilationOptions : CompilationOptions, IEquatable<CS
             StrongNameProvider strongNameProvider = null)
             : this(outputKind, false, moduleName, mainTypeName, scriptClassName, usings, optimizationLevel, checkOverflow, allowUnsafe,
                    cryptoKeyContainer, cryptoKeyFile, cryptoPublicKey, delaySign, platform, generalDiagnosticOption, warningLevel,
+                   specificDiagnosticOptions, concurrentBuild, deterministic,
+                   extendedCustomDebugInformation: true,
```

Comment:
@tmat do you know why we special cased extendedCustomDebugInformation here?  Is it just different than the default on the other ctor?